pkgs/development/python-modules/subprocess-tee/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, pytestCheckHook
+, enrich
+}:
+
+buildPythonPackage rec {
+  pname = "subprocess-tee";
+  version = "0.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01b0z8mcm568v9carhi0py1hs34wrbnlzyvdmhqzipql407kdpk2";
+  };
+
+  propagatedBuildInputs = [
+    setuptools-scm
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    enrich
+  ];
+
+  checkPhase = ''
+    pytest
+  '';
+
```

Comment:
```suggestion
```

Reply:
i am sorry i though i read your request properly. i am doing the change now.